packages/shared/flows.yaml

```diff
@@ -554,6 +554,20 @@ integrations:
         channel_id: string
         user_id: string
         reaction_name: string
+  wildix-pbx:
```

Comment:
Good catch, but this is automatically generated, so it's better to remove this change!

Reply:
Removed